src/fiatExchanges/quotes/FiatConnectQuote.test.ts

```diff
@@ -197,6 +257,7 @@ describe('FiatConnectQuote', () => {
   describe('.getCryptoType', () => {
     it('returns crypto type', () => {
       const quote = new FiatConnectQuote({
+        flow: CICOFlow.CashIn,
         quote: mockFiatConnectQuotes[1] as FiatConnectQuoteSuccess,
         fiatAccountType: FiatAccountType.BankAccount,
       })
```

Comment:
Do we need to change the below assertion to use Currency instead of CryptoType?

Reply:
good catch